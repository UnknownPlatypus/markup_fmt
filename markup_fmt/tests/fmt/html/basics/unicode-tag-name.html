<template>
  <c皮肤
  />
</template>
