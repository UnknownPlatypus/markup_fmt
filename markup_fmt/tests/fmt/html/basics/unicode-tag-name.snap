---
source: markup_fmt/tests/fmt.rs
---
<template>
  <cçš®è‚¤ />
</template>
